ChangeLog.md

```diff
@@ -288,7 +288,7 @@ it possible to access the URI a client instance uses.
 ### Meaningful Exceptions for 4xx and 5xx Responses
 
 `4xx` and `5xx` responses now will result in meaningful exceptions
-being raised. For example, `404` responses will raise `Faraday::Error::ResourceNotFound`.
```

Comment:
We cannot retroactively edit a [change log entry for 0.4.0](https://github.com/ruby-amqp/rabbitmq_http_api_client/blob/master/ChangeLog.md#changes-between-030-and-040)